warehouse/integrations/bigquery/bigquery.go

```diff
@@ -443,44 +441,19 @@ func (bq *BigQuery) loadTable(ctx context.Context, tableName string) (
 	)
 	log.Infon("started loading")
 
-	loadFileLocations, err := bq.loadFileLocations(ctx, tableName)
+	objectLocation, err := bq.uploader.GetSampleLoadFileLocation(ctx, tableName)
 	if err != nil {
-		return nil, nil, fmt.Errorf("getting load file locations: %w", err)
+		return nil, nil, fmt.Errorf("getting sample load file location: %w", err)
 	}
 
-	gcsRef := bigquery.NewGCSReference(warehouseutils.GetGCSLocations(
-		loadFileLocations,
-		warehouseutils.GCSLocationOptions{},
-	)...)
+	gcsRef := bigquery.NewGCSReference(loadFolder(objectLocation))
```

Comment:
Will this change also impact performance?

Reply:
It shouldn't impact performance.